### ISSUE TITLE
Destructible flasks

### ISSUE THREAD

**Issue by DieFluoro** [studio-minus/ppg-bugs-and-requests#1953]
**Feature description**
Flasks should break on impact and any other glass object should eject glass shards and crumble instead of just breaking in half

**Feature purpose**
Realism

**Additional context**
no additional context

**Comment by bibbybabby** [comment]
bottles?

**Comment by DieFluoro** [comment]
do bottles hold liquid?

**Comment by DieFluoro** [comment]
i j figured out that they do but i would still like breakable flasks as a stronger but still destructible alternative

**Comment by bibbybabby** [comment]
> i j figured out that they do but i would still like breakable flasks as a stronger but still destructible alternative

flasks are literally unbreakable unless you mean a separate breakable item

**Comment by DindinYT37** [comment]
> > i j figured out that they do but i would still like breakable flasks as a stronger but still destructible alternative
> 
> flasks are literally unbreakable unless you mean a separate breakable item

And we're back to his original request with this.

**Comment by DieFluoro** [comment]
> > i j figured out that they do but i would still like breakable flasks as a stronger but still destructible alternative
> 
> flasks are literally unbreakable unless you mean a separate breakable item

yes flasks are currently unbreakable, hence my request for them to be breakable. Stronger than bottles, but still breakable.

**Comment by bibbybabby** [comment]
> > > i j figured out that they do but i would still like breakable flasks as a stronger but still destructible alternative
> > 
> > 
> > flasks are literally unbreakable unless you mean a separate breakable item
> 
> yes flasks are currently unbreakable, hence my request for them to be breakable. Stronger than bottles, but still breakable.

This will ouch a lot of contraptions, should be a separate item or context menu thingading
